README.md

```diff
@@ -122,10 +123,10 @@
 * Il capitano raddoppia la somma dei malus ottenuti: **NO**
 
 ### Competizioni
-#### Campionato
+#### Campionato di apertura e di clausura
 * Competizione **a calendario**
-* Struttura dei gironi: **Andata e ritorno più eventuale girone in campo neutro**
-* Inizio: prima giornata successiva all'asta estiva
+* Inizio apertura: prima giornata successiva all'asta estiva
+* Fine clausura: metà delle partite disponibili (es. prima partita 6a di andata del campionato italiano. Rimanendone 32 disponibili, vengono divise in 16 di apertura e 16 di clausura)
 * Fattore campo: **+2** (escluso eventuale ultimo girone neutro)
```

Comment:
girone neutro non ha senso in questo caso

Reply:
done